plugins/inputs/mysql/mysql.go

```diff
@@ -24,6 +30,10 @@ var sampleConfig = `
   #
   # If no servers are specified, then localhost is used as the host.
   servers = ["tcp(127.0.0.1:3306)/"]
+  PerfEventsStatementsDigestTextLimit = 120
```

Comment:
Yes I will add brief explanation of each metric and fix the names of config names.
This config setting are changed frequently, that is why I did not hard coded.
The test fail is because there is metric that tries to gather replication stats, and it only works if the database is slave. I will add the configuration settings to switch it on/off.